contracts/contracts/strategies/sonic/SonicValidatorDelegator.sol

```diff
@@ -148,14 +148,15 @@ abstract contract SonicValidatorDelegator is InitializableAbstractStrategy {
      * This needs to be followed by a `withdrawFromSFC` two weeks later.
      * @param _validatorId The Sonic validator ID to undelegate from.
      * @param _undelegateAmount the amount of Sonic (S) to undelegate.
+     * @return withdrawId The unique ID of the withdrawal request.
      */
     function undelegate(uint256 _validatorId, uint256 _undelegateAmount)
         external
         onlyRegistratorOrStrategist
         nonReentrant
         returns (uint256 withdrawId)
     {
-        return _undelegate(_validatorId, _undelegateAmount);
```

Comment:
Lol, I liked it better before. :D

That said, fine if this style is your preference.